backend/django/core/utils/utils_queue.py

```diff
@@ -376,27 +376,18 @@ def handle_empty_queue(profile, project):
     queue = Queue.objects.get(project=project, type="normal")
     irr_queue = Queue.objects.get(project=project, type="irr")
 
-    queue_count = DataQueue.objects.filter(
-        queue=queue, data__recyclebin__isnull=True
-    ).count()
-    irr_count = DataQueue.objects.filter(
-        queue=irr_queue, data__recyclebin__isnull=True
-    ).count()
+    queue_count = DataQueue.objects.filter(queue=queue).count()
+    irr_count = DataQueue.objects.filter(queue=irr_queue).count()
```

Comment:
Should these two be `DataQueue.objects.unexcluded`?

Reply:
Ah we dont have a DataQueue manager yet... In that case do they need to still have `data__recyclebin__isnull=True`?